### PR TITLE
Add test for pipeline list

### PR DESCRIPTION
**Pull request by sthaha**
# Changes
Adds TestPipelineList test  to validate if `pipeline list -n foo`
returns expected output

Signed-off-by: sthaha@redhat.com

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: pkg/cmd/pipeline/list_test.go [hunk 1]
```diff
@@ -1,24 +1,83 @@
...
-		))
-	test.SeedTestData(test.Data{
-		Pipelines: []*v1alpha1.Pipeline{pipeline},
-	})
-	// NOTE: For now, this servers as a validation that `go test` works fine
-	// and that all dependencies are in place
-
-	// TODO (sthaha) complete the test
+	names := []string{
+		"tomatoes",
```
Comment (hrishin) [inline pkg/cmd/pipeline/list_test.go]: :joy:

Comment (vdemeester) [reply pkg/cmd/pipeline/list_test.go]: :heart:

File: pkg/cmd/pipeline/list_test.go [hunk 2]
```diff
@@ -1,24 +1,83 @@
...
+	"github.com/google/go-cmp/cmp"
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
 	"github.com/tektoncd/pipeline/test"
 	tb "github.com/tektoncd/pipeline/test/builder"
 )
 
+type TestParams struct {
```
Comment (piyush-garg) [inline pkg/cmd/pipeline/list_test.go]: can we put these in some common package to reuse in other tests

Comment (sthaha) [reply pkg/cmd/pipeline/list_test.go]: yes, done!

**Pull request merged**